### ISSUE TITLE
creating own modpack

### ISSUE THREAD

**Issue by darkvip3r** [SKCraft/SKMCLauncher#66]
How do i make a modpack on the new launcher??

**Comment by bardiharborow** [comment]
Hmm... Someone really does need to write an explanation of this. I'll have a look at the code at some stage and get back to you. @sk89q seems to be busy at the moment, but I'm sure he will get back to you if I don't first. =)

**Comment by darkvip3r** [comment]
Yeah its not proplem. Thank you for the reply. Thank you for the help too :)

**Comment by Slind14** [comment]
I would like to know this too.

**Comment by darkvip3r** [comment]
So is it available yet?

**Comment by bendem** [comment]
[README.md](https://github.com/sk89q/skmclauncher/blob/fourth/README.md) : 

> This launcher is no longer maintained due to time constraints. However, the launcher lives on as SKCraft Launcher, but it is purely for sending modpacks to clients. It is no longer a general purpose launcher.

See doc for [SKCraft Launcher](https://github.com/skcraft/launcher) [here](http://confluence.skcraft.com/).
